src/components/Tag/core/tag.module.scss

```diff
@@ -0,0 +1,52 @@
+@use 'sass:map';
+@use '../../../style/variables/colors.scss';
+@use '../../../style/variables/spaces.scss';
+@use '../../../style/mixins/font-config.scss';
+
+.tag {
+  color: map.get(colors.$colors, 'base-white');
+  border-radius: map.get(spaces.$spaces, '1');
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  width: 52px;
+  height: 40px;
+  top: 20px;
+  left: 20px;
```

Comment:
I think top & left props are useless, the code generated by Figma often needs to be adapted to the project. Also the tags should not have a defined height & width. The width will be setted by its content + padding and the height should already be good since the line height (24px) and the padding (8px top / 8px bottom) give you 40px.